oak_utils/src/lib.rs

```diff
@@ -131,46 +131,18 @@ impl prost_build::ServiceGenerator for OakServiceGenerator {
             }
 
             #[allow(dead_code)]
-            pub struct #dispatcher_name<T: #service_name>(T);
+            pub struct #dispatcher_name;
 
             #[allow(dead_code)]
-            impl <T: #service_name> #dispatcher_name<T> {
-                pub fn new(node: T) -> #dispatcher_name<T> {
-                    #dispatcher_name(node)
-                }
-
-                pub fn server_method(method_name: &str) -> Option<&dyn #oak_package::grpc::Invocable<T>> {
+            impl #dispatcher_name {
+                pub fn server_method<T: #service_name>(method_name: &str) -> Option<&dyn #oak_package::grpc::Invocable<T>> {
                     match method_name {
                         #(#dispatcher_invocable_methods,)*
                         _ => None
                     }
                 }
             }
 
-            impl <T: #service_name + Default> Default for #dispatcher_name<T> {
```

Comment:
I don't know why this one and `WithInit` were needed before, but should not they be implemented for `CommandHandler` or `ServerNode` now?

Reply:
No, the beauty of the new approach is that there are no wrappers needed, so it's enough to just implement them on the actual struct that implements the server trait, we don't need to forward them.